python/paddle/fluid/dygraph/dygraph_to_static/convert_call_func.py

```diff
@@ -35,13 +33,71 @@
 from paddle.fluid.dygraph.dygraph_to_static.program_translator import unwrap_decorators
 from paddle.fluid.dygraph.layers import Layer
 
+__all__ = ["convert_call", "not_to_static"]
+
 # TODO(liym27): A better way to do this.
 BUILTIN_LIKELY_MODULES = [
     collections, pdb, copy, inspect, re, six, numpy, logging
 ]
 
 translator_logger = TranslatorLogger()
 
+CONVERSION_OPTIONS = "An attribute for a function that indicates conversion flags of the function in dynamic-to-static."
+
+
+class ConversionOptions(object):
+    """
+    A container for conversion flags of a function in dynamic-to-static.
+
+    Attributes:
+        not_convert(bool): An attribute indicates that the function won't be converted in dynamic-to-static.
+
+    NOTE(liym27): More attributes and methods can be added in this class.
+    """
+
+    def __init__(self, not_convert=False):
+        self.not_convert = not_convert
+
+
+def not_to_static(func=None):
```

Comment:
Should we put `not_to_static` at same place of `to_static`? I think that's more normal.

Reply:
Yes, done. Thanks.